conftest.py

```diff
@@ -227,6 +235,21 @@ def sr_disk(request, host):
             f"disk or block device {disk} is either not present or already used on master host"
     yield disk
 
+@pytest.fixture(scope='session')
+def sr_disk_4k(request, host):
+    disk = request.param
+    if disk == "auto":
+        logging.info(">> Check for the presence of a free 4KiB block device on the master host")
+        disks = host.available_disks(4096)
+        assert len(disks) > 0, "a free disk 4KiB device is required on the master host"
+        disk = disks[0]
+        logging.info(f">> Found free disk device(s) on hostA1: {' '.join(disks)}. Using {disk}.")
+    else:
+        logging.info(f">> Check that disk or block device {disk} is available on the master host")
+        assert disk in host.available_disks(4096), \
+            f"block device {disk} is either not present or already used on master host"
```

Comment:
The assert text should state what is expected, not what the problem is. This problem was already present in `sr_disk`.